plan/main

```diff
@@ -2,3 +2,5 @@
 
 lighttpd
 lighttpd-mod-openssl
+
+wireguard
```

Comment:
We should also install explicitly `wireguard-tools` - which should resolve https://github.com/turnkeylinux/tracker/issues/1824